### ISSUE TITLE
We can't use htsjdk to parse info fields

### ISSUE THREAD

**Issue by tpoterba** [hail-is/hail#2822]
The standard field registrations lead to TERRIBLE behavior -- 

this line in a VCF:
```
##INFO=<ID=MQ0,Number=.,Type=Integer,Description="Number of MAPQ == 0 reads">
```

...gets returned in our code as...
```
INFO=<ID=MQ0,Number=1,Type=Integer,Description="Total Mapping Quality Zero Reads">
```

...leading to match errors when we get to the values.

**Comment by tpoterba** [comment]
also a problem for FORMAT fields

**Comment by tpoterba** [comment]
fixed by #6013
